app/services/delete_work.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+# @abstract Deletes a work, including all of its versions, without regard to its publication data. Note, this service
+# is not intended to be used within the context of the application, but rather as a command-line only method reserved
+# for instances when a specific work needs to be deleted.
+
+class DeleteWork
+  def self.call(uuid)
+    new(uuid: uuid).destroy
+  end
+
+  attr_reader :work
+
+  def initialize(uuid:)
+    @work = Work.find_by(uuid: uuid)
+  end
+
+  def destroy
+    work.versions.map do |version|
+      version.aasm_state = 'withdrawn'
```

Comment:
Is this line necessary? (is never persisted to db)
If it's necessary, is it OK that we're circumventing the state machine API?

Reply:
I put this in 211072880dac71b55656d66ef32aad114962c7e6 that prevents deleting published versions. The whole delete process really needs to be fleshed out a bit more, specifically by fully implementing a withdrawn state. That's ticketed here #805. This PR is to address the short-term need of just deleting something outright, which I get asked to do... and then have to remember to do all these steps.